src/repl/repl_annotations.cpp

```
@@ -394,25 +419,30 @@ void process_str_replace(const StrReplaceAction& action, std::istream& in, std::
       break;
     }
     if (answer == "c" || answer == "copy") {
```

Clang (14.0.6)

[note] +2, including nesting penalty of 1, nesting level increased to 2

(clang)